src/thermo.f90

```diff
@@ -106,8 +106,10 @@ subroutine thermodyn(iunit,A_rcm,B_rcm,C_rcm,avmom_si,linear,atom,sym,molmass, &
    real(wp),parameter :: H = 6.626176D-27 ! PLANCK'S CONSTANT IN ERG-SECONDS
    real(wp),parameter :: AK = 1.3807D-16  ! BOLTZMANN CONSTANT IN ERG/DEGREE
    real(wp),parameter :: conv3 = amutokg*1000 ! 1.6606d-24
-   real(wp),parameter :: magic4 = 2.2868d0 ! unknown
-   real(wp),parameter :: magic5 = 2.3135d0 ! unknown
+   real(wp),parameter :: magic4 = 2.2868d0 ! R*ln(10)/2
+   real(wp),parameter :: magic5 = 2.3135d0 ! R*(ln[(kb/P°)*(2pi * kB * amutokg/h)^(3/2)] + 5/2)
+                                           ! All terms in `magic5` in SI units (excluding R)
+                                           ! P° corresponds to 1 atm in Pascals.
```

Comment:
Thanks for sharing! If you have the correct derivation of the constants, could you implement them into the code to avoid using a fixed float point number?

I'd suggest using the constants in `src/mctc/mctc_constants.f90` and (if possibly necessary for other procedures as well) add the conversion parameters to `src/mctc/convert.f90` to avoid computing these quite extensive constants in the `thermo` module.

Reply:
(Obviously, please check thoroughly if the calculated constants match the currently present fixed numbers, before proceeding.)